extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -36,30 +46,35 @@
 public class TicketValidationService {
 
     /**
+     * Logger for this class.
+     */
+    private static final Logger logger = LoggerFactory.getLogger(TicketValidationService.class);
+
+    /**
      * Service for retrieving CAS configuration information.
      */
     @Inject
     private ConfigurationService confService;
 
     /**
-     * Validates and parses the given ID ticket, returning the username contained
-     * therein, as defined by the username claim type given in
-     * guacamole.properties. If the username claim type is missing or the ID
-     * ticket is invalid, an exception is thrown instead.
+     * Validates and parses the given ID ticket, returning the username
+     * provided by the CAS server in the ticket.  If the
+     * ticket is invalid an exception is thrown.
      *
      * @param ticket
      *     The ID ticket to validate and parse.
+     * @param credentials
```

Comment:
Mind adding the blank line above `@param credentials` to match the style of other comments?